bentoctl_lambda/create_deployable.py

```diff
@@ -1,27 +1,29 @@
 import os
 import shutil
 
+from bentoctl.docker_utils import DOCKERFILE_PATH
+from bentoml._internal.bento.build_config import DockerOptions
+from bentoml._internal.bento.gen import generate_dockerfile
+
 path_to_aws_lambda_files = os.path.join(os.path.dirname(__file__), "./aws_lambda/")
-DOCKERFILE_TEMPLATE = os.path.join(path_to_aws_lambda_files, "Dockerfile.template")
+BENTOML_USER_TEMPLATE = os.path.join(
+    path_to_aws_lambda_files, "bentoctl_user_template.j2"
+)
 APP_FILE = os.path.join(path_to_aws_lambda_files, "app.py")
 
 
-def generate_lambda_deployable(bento_path, bento_metadata, deployable_path):
+def generate_lambda_deployable(bento_path, _, deployable_path):
     # copy bento_bundle to project_path
     shutil.copytree(bento_path, deployable_path)
 
-    # Make docker file with dockerfile template
-    dockerfile = os.path.join(deployable_path, "Dockerfile")
-    with open(DOCKERFILE_TEMPLATE, "r", encoding="utf-8") as f, open(
-        dockerfile, "w"
-    ) as dockerfile:
-        dockerfile_template = f.read()
-        dockerfile.write(
-            dockerfile_template.format(
-                bentoml_version=bento_metadata["bentoml_version"],
-                python_version=bento_metadata["python_version"],
-            )
-        )
+    # Make docker file with user template
+    docker_options_for_lambda = DockerOptions(dockerfile_template=BENTOML_USER_TEMPLATE)
+    dockerfile_generated = generate_dockerfile(
+        docker_options_for_lambda.with_defaults(), use_conda=False
```

Comment:
isn't this going to use the python version on the bentoctl build machine instead of the bento's python version?

Reply:
yep that is a bug. Thanks for pointing it out!
since we have the python version in `bento.yaml` we will use that python version when generating the dockerfile

Reply:
I will take over from here